velox/connectors/hive/HiveConfig.cpp

```diff
@@ -132,7 +132,7 @@ std::string HiveConfig::gcsEndpoint(const Config* config) {
 
 // static
 std::string HiveConfig::gcsScheme(const Config* config) {
-  return config->get<std::string>(kGCSScheme, std::string(""));
+  return config->get<std::string>(kGCSScheme, std::string("https"));
```

Comment:
@majetideepak - just a thought for future, could 'https' be the default for all connectors, and not require exclusive handling?

Reply:
I agree. Google Cloud does not allow insecure connections at all, I think AWS has the same behaviour.

Reply:
S3 API is supported by many more file systems such as Minio and having the HTTP option helps with testing for example since you do not have to set up certificates.
Enabling HTTP or HTTPS is also a part of Presto Hive Configuration for S3. So we need it for S3 to be compatible.

Reply:
There is also no way to make 'https' a default for all connectors since there is no central configuration for this.